language-support/ts/codegen/README.md

```diff
@@ -322,31 +369,32 @@ daml.registerTemplate(Iou);
 ```
 
 `daml2js` has produced:
-  - Type definitions corresponding to the `Transfer` choice, the `Iou` template and the
-    `IouInterface` consisting of its declared choices;
-  - Companion objects for those types;
-  - An `Iou` template associated type definition `Key` in the `Iou` namespace;
-  - A module scoped function call to `registerTemplate` to add the `Iou` object to a central registry.
+
+- Type definitions corresponding to the `Transfer` choice, the `Iou` template and the
+  `IouInterface` consisting of its declared choices;
+- Companion objects for those types;
+- An `Iou` template associated type definition `Key` in the `Iou` namespace;
+- A module scoped function call to `registerTemplate` to add the `Iou` object to a central registry.
 
 #### Daml Interfaces
 
 An `Asset` interface.
 
-```haskell
+````haskell
```

Comment:
looks like some readme changes are wrong. At least this one looks wrong. Maybe I should exclude the readme files from formatting (wasn't my intention to have them formatted too but from what I saw all looked fine. First fault I found so far).